app/views/shared/_navbar.html.erb

```diff
@@ -29,7 +29,9 @@
     <div class="collapse navbar-collapse">
       <ul class="nav navbar-nav navbar-right">
         <% if current_user %>
-          <li><a href="/logout">Logout</a></li>
+          <li>
```

Comment:
Potential Refactor: make links syntax/formatting the same; perhaps refactor all to include helper methods.